composer.json

```diff
@@ -11,18 +11,17 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 | ^8.0",
```

Comment:
Maybe we should make it ^7.3|^8.0 since 7.2 support will end completely at end of Nov.